.github/workflows/on-push-to-release-branch.yaml

```diff
@@ -0,0 +1,63 @@
+name: On push Main
+
+on:
+  push:
+    branches: [release]
+
+jobs:
+  build_csharp:
+    runs-on: ubuntu-latest
+    env:
+      TEST_AUTH_TOKEN: ${{ secrets.ALPHA_TEST_AUTH_TOKEN }}
+      TEST_CACHE_NAME: client-sdk-csharp
+
+    steps:
+      - name: Get current time
+        uses: gerred/actions/current-time@master
+        id: current-time
+
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: "6.0.x"
+
+      - name: Build
+        run: |
+          dotnet build
+        shell: bash
+
+      - name: Unit Test
+        run: |
+          dotnet test tests/Unit/Momento.Sdk.Tests
+        shell: bash
+
+      - name: Integration Test
+        run: |
+          dotnet test tests/Integration/Momento.Sdk.Tests
+        shell: bash
+
+      - name: Incubating Integration Test
+        run: |
+          dotnet test tests/Integration/Momento.Sdk.Incubating.Tests
+        shell: bash
```

Comment:
Not a blocker but is there an easy way in GH actions to dry duplicated steps, eg this block vs what's in ci.yaml

Reply:
it can be done.  pete did some of it in the examples repo, see the `.github/actions` dir.  I also have some re-usable actions in `standards-and-practices`.